package.json

```diff
@@ -114,5 +114,11 @@
   "engines": {
     "node": ">= 6.10"
   },
-  "jest": null
+  "jest": {
```

Comment:
Shouldn't we omit this in clownCallback.js?

Reply:
Fixed